package.json

```diff
@@ -70,6 +70,10 @@
     "ExcessivelySafeCall": {
       "url": "https://github.com/nomad-xyz/ExcessivelySafeCall",
       "commit": "5982c8ea02b84988220de7343e43ebb801069d08"
+    },
+    "solidity-examples": {
+      "url": "https://github.com/LayerZero-Labs/solidity-examples",
```

Comment:
I would remove this dependency and use the interfaces from interfaces/ folder or add the missing function if necessary